src/plugins/footnotes.ts

```diff
@@ -0,0 +1,249 @@
+/*!
```

Comment:
A bit confused here - we still have footnotes directory in the plugins, but the plugins itself not that directory. Can we have it all in that "footnotes" directory?

Reply:
I don't see a point for having the directory, so I'll move the tests out and delete it.

Reply:
Removed.